cartridges/int_bolt_sfra/cartridge/client/default/js/boltProductPageButton.js

```diff
@@ -203,7 +281,7 @@ var wipBuildBoltCartObject = function () {
         return childProducts.length ? JSON.stringify(childProducts) : [];
     };
 
-    var getOptions = function ($productContainer) {
+    var wipGetOptions = function ($productContainer) {
```

Comment:
@dumega should we change the name of this variable back to  `getOptions`?

Reply:
I changed that because the linter was complaining that there was already a getOptions defined (L198). I saw this was part of V's wip version of wipBuildBoltCartObject so assumed it was safe to rename temporarily.

Reply:
Can we change it to something like 'buildOptions', I understand it's part of wip verison code but it seems not a good naming style and might be ignored very easily

Reply:
Yes, good point, I'll update.

Reply:
Fixed.